src/modules/add&remove.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
[OPTIONAL] I can see that you have disabled the linters for import/no-mutable-exports. Kindly avoid doing that since the linters are there to help you achieve best standard practices. Kindly check [this link](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-mutable-exports.md) on how to work around this particular error without disabling the linters.

Reply:
@jerryowusu Thank you so much for the informaion.